samples/MvcSample.Web/project.json

```diff
@@ -17,8 +17,8 @@
     "frameworks": {
         "aspnet50": {
             "dependencies": {
-                "Microsoft.Framework.DependencyInjection.Autofac": "1.0.0-*",
-                "Microsoft.Framework.ConfigurationModel.Json": "1.0.0-*"
+                "Microsoft.Framework.DependencyInjection.Autofac": "1.0.0-beta2",
```

Comment:
Just FYI...this package is not available on public Nuget and as well as the `xunit.runner.kre` and also the `KoreBuild` package.